cli/hse_cli.c

```diff
@@ -970,35 +959,36 @@ cli_hse_kvdb_compact(struct cli_cmd *self, struct cli *cli)
     };
 
     const char *kvdb_home = NULL;
-    uint32_t    timeout_secs = 300;
-    bool        status = false;
-    bool        cancel = false;
-    bool        help = false;
-    int         c;
+    uint32_t timeout_secs = 300;
+    enum kvdb_compact_request request = req_compact;
+    bool help = false;
+    int c, requests = 0;
```

Comment:
Timeout of 300 is way to low in most cases.  Wonder if default should be "no timeout", and let user enable a timeout as an option.

Reply:
I agree with this idea